tree/dataframe/src/RSqliteDS.cxx

```diff
@@ -132,10 +430,10 @@ RSqliteDS::RSqliteDS(std::string_view fileName, std::string_view query)
 RSqliteDS::~RSqliteDS()
 {
    // sqlite3_finalize returns the error code of the most recent operation on fQuery.
-   (void) sqlite3_finalize(fQuery);
+   (void)sqlite3_finalize(fQuery);
```

Comment:
is the `(void)` cast needed? i.e. do these kind of ignored return values trigger compiler warnings? (can't see it e.g. [here](https://godbolt.org/z/La9zRq))